muthash/nbo-java-devs
# #155768857 Connect RecyclerView to MainActivity

**What does this PR do?**
Connects the RecyclerView to the main activity of the Java Developers App

**Description of Task to be completed?**
- Connect the RecycleView to the main activity.
- Set up continuous integration with CircleCI.

**How should this be manually tested?**
- Clone the repository and set up the Android Studio as described on the README
- Run the application
- The app should show the screen as shown on the screenshot below

**What are the relevant pivotal tracker stories?**
##155738450, ##155768857, #155768911

**Screenshots**
<img width="294" alt="screen shot 2018-08-06 at 17 45 27" src="https://user-images.githubusercontent.com/33781312/43723559-9a95f880-99a0-11e8-831b-3ec390d52ce9.png">
